base/sparse/sparsevector.jl

```diff
@@ -607,26 +607,28 @@ getindex(x::AbstractSparseVector, ::Colon) = copy(x)
 
 ### show and friends
 
-function showarray(io::IO, x::AbstractSparseVector;
-                   header::Bool=true, repr=false)
-
+function show(io::IO, x::AbstractSparseVector)
     n = length(x)
     nzind = nonzeroinds(x)
     nzval = nonzeros(x)
     xnnz = length(nzind)
 
-    if header
+    if get(io, :multiline, false)
         println(io, summary(x))
```

Comment:
nevermind, I was confused here. I just meant to note that this method also this method also doesn't support `:multiline` correctly (due to `sep` below)